File: padaria-app/src/app/module/models/src/employee/level-schooling.ts
import { describe } from '@module/utils/functions/enum';

export enum LevelSchooling {
  None = 0,
  IncompleteHigherEducation = 1,
  CompleteHigherEducation = 2,
  IncompleteSchoolLevel = 3,
  CompleteSchoolLevel = 4,
  PostGraduation = 5,
}

describe(LevelSchooling, {
  None: 'Nenhum',
  IncompleteSchoolLevel: 'Enisno Médio Incompleto',
  CompleteSchoolLevel: 'Enisno Médio Completo',
  IncompleteHigherEducation: 'Ensino Superior Incompleto',
  CompleteHigherEducation: 'Ensino Superior Completo',
  PostGraduation: 'Pós-Graduação',
});

File: padaria-api/src/main/java/com/padaria/model/levelSchooling/LevelSchooling.java
package com.padaria.model.levelSchooling;

public enum LevelSchooling {

    None  ("Nenhum"),
    IncompleteHigherEducation ("Enisno Médio Incompleto"),
    CompleteHigherEducation ("Enisno Médio Completo"),
    IncompleteSchoolLevel ("Ensino Superior Incompleto"),
    CompleteSchoolLevel ("Ensino Superior Completo"),
    PostGraduation ("Pós-Graduação");

    private final String description;

    LevelSchooling(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}
